assets/wizards/dashboard/index.js

```diff
@@ -38,4 +79,4 @@ class Dashboard extends Component {
 		);
 	}
 }
-render( <Dashboard items={ newspack_dashboard } />, document.getElementById( 'newspack' ) );
+render( <Dashboard items={ newspack_dashboard } />, document.getElementById( 'newspack' ) ); // eslint-disable-line
```

Comment:
Nitpick: instead of disabling linter, `newspack_dashboard` should be defined as global at the top of the file:

```
/* global newspack_dashboard */
```

Reply:
Done in https://github.com/Automattic/newspack-plugin/pull/496/commits/a327fe8996c9d3c8b31d1a9d0082b15f1c5c66a9